scripts/release_train.py

```diff
@@ -0,0 +1,224 @@
+#!/usr/bin/env python3
+
+"""
+# Release train script
+
+This module contains a CLI script for unleashing the charm release bump train.
+For all input charms, the script will:
+- grab whatever revision is on edge and release it to beta
+- grab whatever revision is on beta and release it to candidate
+- grab whatever revision is on candidate and release it to stable
```

Comment:
Curious how this works without the track (`latest`) being specified (hard-coded) anywhere.